base/setup/lib/muilanguages.h

```diff
@@ -698,7 +698,7 @@ const MUI_LANGUAGE MUILanguageList[] =
     {L"00000415", L"1250", L"852", L"10029", L"Polish", L"191", LatinFonts, plPLLayouts},
 #endif
 #ifdef LANGUAGE_PT_PT
-    {L"00000816", L"1252", L"850", L"10000", L"Portuguese (Portugal)", L"193", LatinFonts, ptPTLayouts},
+    {L"00000816", L"1252", L"850", L"10000", L"Português (Portugal)", L"193", LatinFonts, ptPTLayouts},
```

Comment:
I'm not sure this needs to be translated. If you look through this file, all languages are in English.

Reply:
This is what shows up in the main language list in the usetup, and all language names should be displayable in any currently-selected code page.